src/CC0-1.0.xml

```diff
@@ -143,6 +143,6 @@
             </list>
         </item>
       </list>
-    
+      <alt><p>For more information, please see &lt;http://creativecommons.org/publicdomain/zero/1.0/&gt;</p></alt>
```

Comment:
Does `<alt>` work without a `name` and `match`?  And while I think we want to match files which include this line, I don't think we want to include it in our canonical text.  So I still prefer the [phrasing I'd suggested here][1].

[1]: https://github.com/spdx/license-list-XML/pull/489#discussion_r153060299

Reply:
OK, I didn't fully understand -- it's including text in a `match` that keeps it out of canonical text, not wrapping it in `<alt>` -- I'll go with your suggestion, thanks for explaining.